build fail is real (queries need to be updated)
```
======================================================================
FAIL: custom.icds_reports.utils.aggregation_helpers.tests.tests:TestQueryDiffs.test_agg_sql_diff
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/utils/aggregation_helpers/tests/tests.py", line 67, in test_agg_sql_diff
    """)
AssertionError: 
-------------------- Diffs detected for agg helper 'agg-awc' --------------------
DISTRIBUTED DIFF
=================
--- master
+++ branch
@@ -139,7 +139,7 @@
         WHERE (opened_on <= %(end_date)s AND
               (closed_on IS NULL OR closed_on >= %(start_date)s ))
 
-        GROUP BY ucr.awc_id, ucr.supervisor_id) ut
+        GROUP BY ucr.supervisor_id, ucr.awc_id) ut
         WHERE ut.awc_id = agg_awc.awc_id and ut.supervisor_id=agg_awc.supervisor_id;
         
 {"end_date": "2019-01-31T23:59:59", "month_end_11yr": "2008-01-31T23:59:59", "month_end_15yr": "2004-01-31T23:59:59", "month_start_14yr": "2005-01-01T00:00:00", "month_start_15yr": "2004-01-01T00:00:00", "month_start_18yr": "2001-01-01T00:00:00", "start_date": "2019-01-01T00:00:00"}
To fix these diffs:
    1. Make sure you have applied changes to both the helper classes.
    2. Run "python manage.py update_icds_aggregation_output" and commit the changes.
```

--- a/custom/icds_reports/utils/aggregation_helpers/tests/sql_output/agg-awc.distributed.txt
+++ b/custom/icds_reports/utils/aggregation_helpers/tests/sql_output/agg-awc.distributed.txt
@@ -139,7 +139,7 @@
         WHERE (opened_on <= %(end_date)s AND
               (closed_on IS NULL OR closed_on >= %(start_date)s ))
 
-        GROUP BY ucr.awc_id, ucr.supervisor_id) ut
+        GROUP BY ucr.supervisor_id, ucr.awc_id) ut
         WHERE ut.awc_id = agg_awc.awc_id and ut.supervisor_id=agg_awc.supervisor_id;
         
 {"end_date": "2019-01-31T23:59:59", "month_end_11yr": "2008-01-31T23:59:59", "month_end_15yr": "2004-01-31T23:59:59", "month_start_14yr": "2005-01-01T00:00:00", "month_start_15yr": "2004-01-01T00:00:00", "month_start_18yr": "2001-01-01T00:00:00", "start_date": "2019-01-01T00:00:00"}

